Testing/establish_test_directory.md

```diff
@@ -0,0 +1,73 @@
+# Establish Testing Directory
+
+Navigate to the application's sencha-workspace folder and make a directory called tests:
+```
+cd /path/to/sencha-workspace/
+mkdir tests
+```
+
+You should now have a tests folder in your app's sencha-workspace. This folder will serve as the root of all our testing code and the location of the siesta framework. Download the latest version of Siesta-lite library here: [www.bryntum.com/products/siesta/download-lite/](www.bryntum.com/products/siesta/download-lite/)
```

Comment:
@aliyeysides `sencha-workspace` is our internal convention for naming the `Sencha Cmd Workspace` directory within our repositories. This guide is intended to be a general guide for all Ext JS users, not just Jarvus staff or those following Jarvus' conventions outside Ext JS. While `sencha-workspace` is a good name for the directory in your example code, in copy you should be careful to refer to the feature using the same language/capitalization as the upstream documentation, and link to that the first time you use the term.

> Navigate to your application's [Sencha Cmd Workspace](https://docs.sencha.com/cmd/6.x/workspaces.html)

Also, be consistent with use of uses of abbreviations like app/application. Generally it's best to avoid abbreviating things in writing